src/components/ComponentPreview.js

```diff
@@ -3,55 +3,48 @@ import PropTypes from 'prop-types';
 import IframeResizer from 'iframe-resizer-react';
 import DevDocLinks from './DevDocLinks';
 
-class ComponentPreview extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const ComponentPreview = ({
+    componentName,
+    hasReact,
+    hasAngular,
+    hasHTML,
+    componentType,
+    minHeight,
+    maxHeight,
+    maxWidth,
+    allowScrolling,
+  }) => {
```

Comment:
This section is indented one too many.

Reply:
I've made the changes you've asked for. And I updated my staging with upstream too.